crates/sui-prover/tests/snapshots/prettyprint-struct.move.snap

```diff
@@ -12,11 +10,3 @@ error: prover::ensures assertion does not hold
    │     ^^^^^^^^^^^^^^^^^^^
    │
    =     at tests/inputs/prettyprint-struct.move:19: foo_spec
-   =         x = 0u64
-   =     at tests/inputs/prettyprint-struct.move:20: foo_spec
-   =         x = 0u64
-   =     at tests/inputs/prettyprint-struct.move:14: foo
-   =         result = foo.MyStruct{a = 0u64, b = 0u64}
-   =     at tests/inputs/prettyprint-struct.move:16: foo
-   =         res#1#0 = foo.MyStruct{a = 0u64, b = 0u64}
```

Comment:
let's add `log::var(res)` to the test to check the pretty-printing of the `struct`